src/main/java/github/weichware10/util/config/ConfigLoader.java

```diff
@@ -4,6 +4,8 @@
 import com.fasterxml.jackson.databind.DatabindException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import github.weichware10.util.Logger;
+import github.weichware10.util.db.DataBaseClient;
+
 import java.io.File;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.io.File'